source/sass/buyers-guide/views/product.scss

```
@@ -1,5 +1,21 @@
 $pni-product-breakpoint-larger: $bp-md;
 
+#view-product-page {
+  #multipage-nav-mobile {
+    margin-bottom: 0;
+  }
+
+  .multipage-link.active {
+    color: $gray-60;
+    border-bottom: none;
+
+    &:hover {
+      color: $black;
+      border-bottom: 3px solid $black;
+    }
+  }
```

```suggestion
  .multipage-link.active {
    color: $gray-60;
    border-bottom: none;
  }
```

works fine to me without the &:hover override

---

are you sure? because without the hover, you don't get the black, underline highlight for "all" that you get for all the other categories when you mouse over.

---

turns out this is needed